src/httpserver/handlers/head.clj

```diff
@@ -0,0 +1,9 @@
+(ns httpserver.handlers.head
+  (:require [httpserver.handlers.two-hundred :as two-hundred]
+            [httpserver.handlers.four-zero-four :as four-zero-four]))
+
+(defn call [request]
+  (cond
+    (= "/" (:path request)) (two-hundred/call request)
```

Comment:
After doing the partial content test and writing the clojure file_io stuff. I will make another PR to change this to instead use the file_io to check if the path is valid or invalid when deciding to return either a 200 or a 404.